.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+
+### Checklist
+
+- [ ] Changes have been tested
+- [ ] `Change-type` present on at least one commit
+- [ ] `Changelog-entry` present on at least one commit
```

Comment:
We can have this as optional. Because it defaults to summary. We should mention this.

Reply:
As please add that the commits need to comply with: http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines . See the example there.